runexporterapp.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/bash
 
-cd /ironic-prometheus-exporter
+cd /usr/lib/python2.7/site-packages/ironic_prometheus_exporter/
 export IRONIC_CONFIG=/etc/ironic/ironic.conf
 export FLASK_RUN_HOST=0.0.0.0
```

Comment:
Side note: we may need to make this customizable eventually

Reply:
Agree